src/system/services/cognitive/sentimentService.ts

```diff
@@ -0,0 +1,66 @@
+import { injectable, inject } from "inversify";
+import * as serviceContracts from "../../contract/contracts";
+
+import { IConfig } from "../../contract/systemEntities";
+import { serviceBase } from '../../services/serviceBase'
+import * as systemContracts from '../../contract/contracts';
+
+@injectable()
```

Comment:
Please ad some JS docs to describe this service, and link to any relevant doco

Reply:
Hey Az, this one still doesn't have an example on how to use it or any doco... I don't want random un-associated things in the repo if possible pleased.